<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceInfoWindow</class>
 <widget class="QWidget" name="DeviceInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color:#99dddddd;
}</string>
  </property>
  <widget class="QWidget" name="InfoPage" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>861</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color:#00000000;
}</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>861</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
	border:0px outset #cccccc;
	border-radius:00px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>861</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
	border:0px outset #cccccc;
	border-radius:50px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>611</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#990000ff;
	color:#ffffff;
	border:10px outset #ff0000ff;
	border-radius:50px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="closePageButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>15</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleTextarea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>861</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777214</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	color:#ffffff;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="deviceFunctionInfoText">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>301</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:0px outset #ff0000ff;
	border-radius:50px;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="deviceIcon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteDevButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#990000ff;
	color:#ffffff;
	border:1px outset #ff0000ff;
	border-radius:50px;
}
QPushButton::hover{
	background-color:#99000099;
	
}
QPushButton::pressed{
	color:#999999;
}</string>
    </property>
    <property name="text">
     <string>删除该设备</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>60</y>
      <width>20</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
}</string>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>设备类型文档</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>类型:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DeviceTypeText">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DeviceName</string>
    </property>
   </widget>
   <widget class="QWidget" name="deviceAddress" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>占用地址:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="useMemBegin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:1px outset #ffaaaaaa;
	border-radius:50px;
}</string>
     </property>
     <property name="text">
      <string>0x00000000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="useMemEnd">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0x00000000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>功能:</string>
    </property>
   </widget>
   <widget class="QWidget" name="InterNum" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>中断号:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="interNumTextArea">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:1px outset #ffaaaaaa;
	border-radius:50px;
}</string>
     </property>
     <property name="text">
      <string>000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
}</string>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifyDataButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#990000ff;
	color:#ffffff;
	border:1px outset #ff0000ff;
	border-radius:50px;
}
QPushButton::hover{
	background-color:#99000099;
	
}
QPushButton::pressed{
	color:#999999;
}</string>
    </property>
    <property name="text">
     <string>修改设备内部数据</string>
    </property>
   </widget>
   <zorder>titleArea</zorder>
   <zorder>label_3</zorder>
   <zorder>titleTextarea</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>closePageButton</zorder>
   <zorder>deviceAddress</zorder>
   <zorder>deviceIcon</zorder>
   <zorder>deviceFunctionInfoText</zorder>
   <zorder>DeviceTypeText</zorder>
   <zorder>label_7</zorder>
   <zorder>line</zorder>
   <zorder>DeleteDevButton</zorder>
   <zorder>InterNum</zorder>
   <zorder>line_2</zorder>
   <zorder>ModifyDataButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
